### PR TITLE
LPS-198099 Inline UserPersonalSitePermissions methods into CompanyModelListener

### PR DESCRIPTION
**Pull request by MrJohn1911**
Ticket: https://liferay.atlassian.net/browse/LPS-198099

### CODE REVIEW

**Comment by catarineaguiar** [comment]
relevant passed in the last pull, running echo suite to make sure it doesn't have regressions

**Comment by catarineaguiar** [comment]
ci:test:echo

**Comment by Kvale1733** [comment]
@MrJohn1911 No regressions found.

File: modules/apps/application-list/application-list-user-personal-site-permissions/src/main/java/com/liferay/application/list/user/personal/site/permissions/internal/UserPersonalSitePermissions.java [hunk 1]
```diff
@@ -1,278 +0,0 @@
...
-	}
-
-	@Activate
-	protected void activate(BundleContext bundleContext) {
-		_bundleContext = bundleContext;
-
-		String filter = StringBundler.concat(
-			"(&(objectClass=", PanelApp.class.getName(),
-			")(panel.category.key=", PanelCategoryKeys.SITE_ADMINISTRATION,
-			"*))");
```
Comment (shuyangzhou) [inline modules/apps/application-list/application-list-user-personal-site-permissions/src/main/java/com/liferay/application/list/user/personal/site/permissions/internal/UserPersonalSitePermissions.java:90]: @MrJohn1911 this change can not be right. The original filter here is a "*" wildcard. After your change the wildcard is gone, you won't be collecting the same thing anymore.

**Pull request closed without merging**